maestro-orchestra-models/src/main/java/maestro/orchestra/Commands.kt

```diff
@@ -423,6 +423,25 @@ data class AssertWithAICommand(
     }
 }
 
+data class AssertVisualCommand(
+    val baseline: String,
+    val thresholdPercentage: Int,
+    override val optional: Boolean = false,
+    override val label: String? = null,
+) : Command {
+    override fun description(): String {
+        return label ?: "Assert visual difference with baseline $baseline (threshold: $thresholdPercentage%)"
```

Comment:
It's a nitpick, I know, but thinking of terminal in a vscode window on a MacBook... Could this be shorter?

Perhaps not 🤔

Reply:
- `visual difference` -> `visual diff` maybe?
- display just filename of $baseline, not the whole path